### PR TITLE
refactor(editor): Patch ElementUI tooltip memory leak

### PR DESCRIPTION
**Pull request by OlegIvaniv**
ElementUI tooltip doesn't correctly remove all the events watchers when it gets destroyed. This leads to hanging detached elements and event listeners. This is especially bad for the node creator panel because each `NodeItem` contains two `NodeIcon`, which contain `n8n-tooltip`(wrapped ElementUI tooltip), resulting in thousands of detached DOM elements.

With this change, it should be safe to run the tests to check for all the rendered nodes. But I also improved it to be more. efficient by getting a list of node names and comparing list items instead of querying dom for every node item.

### CODE REVIEW

**Review by cstuncsik — APPROVED** [review]

File: patches/element-ui@2.15.12.patch [hunk 1]
```diff
@@ -0,0 +1,263 @@
+diff --git a/lib/element-ui.common.js b/lib/element-ui.common.js
```
Comment (netroy) [inline patches/element-ui@2.15.12.patch:1]: Does it makes sense to send this patch as an PR to upstream,so that we don't have to maintain this?

Comment (netroy) [reply patches/element-ui@2.15.12.patch:1]: Is this patch from [this PR](https://github.com/ElemeFE/element/pull/20671)?

Comment (OlegIvaniv) [reply patches/element-ui@2.15.12.patch:1]: Yes

Comment (netroy) [reply patches/element-ui@2.15.12.patch:1]: [Looking at the releases, they are actively merging into 2.x](https://github.com/ElemeFE/element/releases).

Comment (OlegIvaniv) [reply patches/element-ui@2.15.12.patch:1]: Maybe one day :D

Comment (netroy) [reply patches/element-ui@2.15.12.patch:1]: I haven't actually tested this yet. but, since @cstuncsik approved it, please feel free to merge.

**Comment by janober** [comment]
Got released with `n8n@0.205.0`

**Pull request merged**